### ISSUE TITLE
Import failing from non-existent file

### ISSUE THREAD

**Issue by NotNite** [putoutjs/minify#1]
In package.json, the exports are defined as such:

```json
{
  "exports": {
    ".": "./bundle/minify.js"
  }
}
```

However, only the minified version exists on the filesystem after install:

![image](https://github.com/putoutjs/minify/assets/44414597/9458a610-8bf3-434a-9947-b4b35f3046fc)

Meaning import fails:

```shell
$ cat test.mjs
import { minify } from "@putout/minify";

const out = minify(`function test() {
  console.log("test");
}
test();`);
console.log(out);

$ node test.mjs
node:internal/errors:484
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'D:\code\js\minify-test\node_modules\@putout\minify\bundle\minify.js' imported from D:\code\js\minify-test\test.mjs
    at new NodeError (node:internal/errors:393:5)
    at finalizeResolution (node:internal/modules/esm/resolve:260:11)
    at moduleResolve (node:internal/modules/esm/resolve:879:10)
    at defaultResolve (node:internal/modules/esm/resolve:1087:11)
    at nextResolve (node:internal/modules/esm/loader:161:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:831:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:413:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

Editing the package.json seems to fix it, but of course is not a proper solution:

```shell
$ sed -i "s|./bundle/minify.js|./bundle/minify.min.js|"  node_modules/@putout/minify/package.json
$ node test.mjs
function test(){console.log("test");}test();
```

**Comment by coderaiser** [comment]
Thank you! Fixed in [v1.13.1](https://github.com/putoutjs/minify/releases/tag/v1.13.1) 🎉 . Is it works for you?

**Comment by NotNite** [comment]
Works, thanks!
